src/test/run-make/emit-path-unhashed/Makefile

```diff
@@ -0,0 +1,26 @@
+-include ../../run-make-fulldeps/tools.mk
+
+OUT=$(TMPDIR)/emit
+
+# --emit KIND=PATH should not affect crate hash vs --emit KIND
```

Comment:
Can you check that adding or removing `--emit` kinds does change the crate hash?

Reply:
OK, done.